### PR TITLE
Update course setup instructions

### PR DESCRIPTION
**Pull request by eloots**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,40 +14,137 @@ Through a series of exercises, we look at specific Scala 3 features and apply th
...
+ 12.      exercise_011_multiversal_equality
+ 13.      exercise_020_opaque_type_aliases_alt
+ 14.      exercise_021_multiversal_equality
+```
+
+As can be seen in the above output, `cmtc` installed the course in the `/Users/ericloots/Library/Caches/com.lunatech.cmt/Courses/lunatech-scala2-to-scala3-course` folder. Note that this location is OS specific, so, take a note of the location.
+
+### Loading the course exercises
+
+You can now load the course exercises in the development environment of your choice. You will be
```
Comment (adpi2) [inline README.md:103]:
```suggestion
You can now load the course exercises in the development environment of your choice. You will be
```

What does it mean to load the course in the development environment of your choice?

Comment (eloots) [reply README.md:103]: I mean the IDE of choice (IntelliJ or VSCode with metals, or ...)

Comment (adpi2) [reply README.md:103]:
```suggestion
You can now load the course exercises in the development environment of your choice (IntelliJ, VSCode with Metals, etc). You will be
```

File: exercises/exercise_001_dotty_deprecated_syntax_rewriting/IDE-Setup.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Installing IntelliJ with the Scala Language Plugin
+
+* [Install the IntelliJ IDE](https://www.jetbrains.com/help/idea/installation-guide.html#standalone) (Community or Ultimate edition)
+    * [Install the Scala Language](https://www.jetbrains.com/help/idea/discover-intellij-idea-for-scala.html)
+
+#### Installing Visual Code Studio with the Metals extension
+
+* [Install Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+    * [Install the Visual Code Studio Metals extension](https://scalameta.org/metals/docs/editors/vscode/)
```
Comment (adpi2) [inline exercises/exercise_001_dotty_deprecated_syntax_rewriting/IDE-Setup.md:22]: We should probably mention that Metals is the *Scala* extension.

```suggestion
    * [Install Metals, the Scala extension](https://scalameta.org/metals/docs/editors/vscode/)
```

**Review by adpi2 — APPROVED** [review]
One minor comment, but apart from that it is looking good.

File: exercises/exercise_001_dotty_deprecated_syntax_rewriting/IDE-Setup.md (hunk 2 above)
Comment (adpi2) [reply exercises/exercise_001_dotty_deprecated_syntax_rewriting/IDE-Setup.md]: This was actually repeated in all the exercises.

**Pull request merged**